resources/build/increment-version.sh

```diff
@@ -49,8 +49,8 @@ if [[ $# -gt 0 ]]; then
   fi
 
   if [[ $action == commit ]]; then
-    if [ "$base" != "master" ] && [ "$base" != "beta" ]; then
-      echo "Invalid branch $base specified: must be master or beta."
+    if [ "$base" != "master" ] && [ "$base" != "beta" ] && [[ ! "$base" =~ ^stable-[0-9]+\.[0-9]+$ ]]; then
```

Comment:
I thought a long while ago, you wished for all future `stable` builds to just be on the branch `stable`.
We we sticking to `stable-14.0` for now?

Reply:
I think we stick with `stable-14.0` because that gives us the ability (at least in theory) to build older versions and track them. They are versions we care about 😁